inlong-manager/manager-service/pom.xml

```diff
@@ -70,10 +70,6 @@
             <scope>provided</scope>
         </dependency>
 
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
```

Comment:
Please do not change this file when you rebase from the master branch, thanks.

Reply:
fix it